### PR TITLE
GsoC 2022 Introductory Blog

### PR DESCRIPTION
**Pull request by Mashijams**

### CODE REVIEW

File: content/blog/mashijams/2022-05-21_gsoc_22__xfs_filesystem_support.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+As of now we have read support for xfs version 4.
+
+Inodes, short directories, leaf directories, and node directories can be read successfully.
+
+I also did some testing for xfs during my application period, details for which can be found in this forum [post](https://discuss.haiku-os.org/t/xfs-file-system-testing/12094)
+
+# Project Plan : 
+
+I will begin my project by adding xfs version 5 read support, Its long time since version 5 support is available on linux so it will be good if we have same for Haiku as well.
```
Comment (humdingerb) [inline content/blog/mashijams/2022-05-21_gsoc_22__xfs_filesystem_support.md:29]:
```suggestion
I will begin my project by adding xfs version 5 read support. Linux has had it for a long time, so Haiku needs to support it, too.
```

File: content/blog/mashijams/2022-05-21_gsoc_22__xfs_filesystem_support.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- Test if we are getting valid version 5 superblock on mounting filesystem image and fix bugs/issues that comes.
+
+After this I believe we should be able to detect valid super block for both xfs version 4 & 5.
+
+# Conclusion :
+
+I will be very active on this [Forum](https://discuss.haiku-os.org/t/gsoc-22-xfs-project-progress/12205) Thread where I will post small updates about my project and take help from mentors.
+
+I will also make blogs frequently to give community Idea about my work done and upcoming plans for xfs project.
```
Comment (humdingerb) [inline content/blog/mashijams/2022-05-21_gsoc_22__xfs_filesystem_support.md:47]:
```suggestion
I will also make blogs frequently to update the community about my progress and upcoming plans for the xfs project.
```

**Review by humdingerb — COMMENTED** [review]
Just a few grammatical nit picks... :)

File: content/blog/mashijams/2022-05-21_gsoc_22__xfs_filesystem_support.md (hunk 2 above)
Comment (nephele-gh) [reply content/blog/mashijams/2022-05-21_gsoc_22__xfs_filesystem_support.md]: Should probably replace "blogs" with "blog posts" or "blog entries"

Comment (Mashijams) [reply content/blog/mashijams/2022-05-21_gsoc_22__xfs_filesystem_support.md]: Done

**Pull request merged**